src/fields/FileField.h

```diff
@@ -0,0 +1,31 @@
+#ifndef FILEFIELD_H
+#define FILEFIELD_H
+
+#include "Field.h"
+#include <iostream>
+#include <fstream>
+#include <stdio.h>
+
+using namespace std;
+
+class FileField: public Field {
+
+    private:
+        string _fieldValue;
+        string _defaultValue;
+        string _folderPath;
+
+    public:
+        FileField(string fieldName, string folderPath="", string defaultValue="", bool isPrimaryKey=false, bool isUnique=false, bool isNullable=true, bool isAutoFilled=false);
+
+        void setValue(string value);
```

Comment:
Add a `setContentStream` method:
```
void setContentStream(ifstream & fileStream);
```

You would need a private variable for the `fileStream`. This would then get used to in private method I spoke of above to save the file.